docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.hero.scss

```diff
@@ -92,6 +91,7 @@
 
   &__text {
     display: block;
+    align-self: center;
```

Comment:
This fixes an autofixer error. 
<img width="850" alt="autoprefixer error" src="https://user-images.githubusercontent.com/12678977/92784546-ac423f00-f374-11ea-9740-56c14a5c38c6.png">